src/components/index.js

```diff
@@ -1,6 +1,7 @@
 export { default as Anchors } from './anchors'
 export { default as Category } from './category'
 export { default as Preview } from './preview'
+export { default as NotFound } from './not-found'
```

Comment:
Do we need to export this component?
Seems strange that many of these exports are unused. Only usage I can see is in `src/index.js`, but that's just 

```
  Picker,
  NimblePicker,
  Emoji,
  NimbleEmoji,
  Category,
```

Reply:
Not sure if we need to export it or not but everything breaks if I don't export it so keeping it for now unless there's some obvious workaround that I don't know?

Reply:
It will be because you are `import { NimbleEmoji, NotFound } from '.'` in the Category file, which is importing from the current directory which looks in the directories `index.js` file to file the component

So `import { NimbleEmoji, NotFound } from '.'`  is the same as `import { NimbleEmoji, NotFound } from './index.js'` which is why you have to export the NotFound component to be able to be imported